mizar/proto/mizar/proto/interface.proto

```diff
@@ -36,6 +35,8 @@ enum InterfaceStatus {
   init = 0;
   queued = 1;
   consumed = 2;
+  dequeued = 3; 
+  deleted = 4;
```

Comment:
Since you clean-up, the objects from the interfaces table, Do we really need these status values?